modules/yieldlabBidAdapter.js

```diff
@@ -441,4 +445,31 @@ function extractSizes(bid) {
   return Array.from(deduplicatedSizeStrings)
 }
 
+/**
+ * Gets the floor price if the Price Floors Module is enabled for a given auction,
+ * which will add the getFloor() function to the bidRequest object.
+ *
+ * @param {Object} bid
+ * @param {string[]} sizes
+ * @returns The floor CPM of a matched rule based on the rule selection process (mediaType, size and currency),
+ *          using the getFloor() inputs. Multi sizes and unsupported media types will default to '*'
+ */
+function getBidFloor(bid, sizes) {
+  if (!isFn(bid.getFloor)) {
+    return undefined;
+  }
+  const mediaTypes = deepAccess(bid, 'mediaTypes');
+  const mediaType = mediaTypes !== undefined && spec.supportedMediaTypes.includes(Object.keys(mediaTypes)[0].toLowerCase())
```

Comment:
The result of `Object.keys(mediaTypes)[0].toLowerCase()` could be stored in a variable, otherwise it might get executed twice.